ci/jenkins/test.sh

```diff
@@ -513,7 +513,7 @@ function run_e2e {
     if [[ $FLEXIBLE_IPAM == true ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
     else
-        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus
+        go test -v --run=TestMulticast antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus
```

Comment:
Better not limit the test cases only with "Multicast". We need to also check if the general cases are broken when Multicast feature gate is on.

Reply:
Yes, will delete this and run whole test cases before merging. Use this for quick quick for now.